components/resultscreen/index.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import {View, Text} from 'react-native';
+import stylesscreen from '../middlescreen/styles';
+
+const PantallaResultado = (props) => {
+  return (
+    <View style={stylesscreen.pantallaCalculadora}>
+      <Text style={stylesscreen.textooperacion}>{props.textoOperacion}</Text>
+
+      <Text style={stylesscreen.textoresultado}>{props.textoResultado}</Text>
+    </View>
+  );
+};
+
+export {PantallaResultado};
```

Comment:
Para componentes solemos usar default exports. Y a la carpeta la llamamos igual que al componente.

```suggestion
export default PantallaResultado;
```

 Despues lo importas así
:import PantallaResultado from './componentes/PantallaResultado'